schema/omega-services.async.api.yaml

```diff
@@ -777,6 +837,195 @@ components:
+        - current-description
+        - description
+      properties:
+        identifier:
+          $ref: '#/components/schemas/GeneralIdentifier'
+        type:
+          $ref: '#/components/schemas/RecordType'
+        creator:
+          type: array
+          items:
+            $ref: '#/components/schemas/GeneralIdentifier'
+          description: the identifier of the Creating Body of the Record concept, i.e. a URI to an Agent
+        current-description:
+          $ref: '#/components/schemas/GeneralIdentifier'
+          description: the identifier of the current description of the Record concept
+        description:
+          type: array
+          items:
+            $ref: '#/components/schemas/RecordDescriptionFull'
+
+    RecordDescriptionSummary:
+      type: object
+      required:
+        - identifier
+        - label
+        - abstract
+        - access-rights
+        - is-part-of
+        - version-timestamp
+      properties:
+        identifier:
+          $ref: '#/components/schemas/GeneralIdentifier'
+        label:
+          $ref: '#/components/schemas/GeneralLabel'
+        abstract:
+          $ref: '#/components/schemas/GeneralDescription'
+        access-rights:
+          $ref: '#/components/schemas/GeneralIdentifier'
+          description: the identifier of the Rights Statement for the Record description
+        is-part-of:
+          type: array
+          items:
+            $ref: '#/components/schemas/GeneralIdentifier'
+          description: Links to the parent of which this is part of
+        previous-sibling:
+          $ref: '#/components/schemas/GeneralIdentifier'
+          description: Links to the previous sibling. Used when there is an order of resources to be applied across siblings
+        version-timestamp:
+          $ref: '#/components/schemas/VersionTimestamp'
+        previous-description:
+          $ref: '#/components/schemas/GeneralIdentifier'
+
+    RecordDescriptionFull:
+      type: object
+      allOf:
+        - $ref: '#/components/schemas/RecordDescriptionSummary'
+        - $ref: '#/components/schemas/RecordDescriptionProperties'
+
+    RecordDescriptionProperties:
+      type: object
+      properties:
+        secondary-identifier:
+          type: array
+          items:
+            $ref: '#/components/schemas/GeneralTypedIdentifier'
+        asset-legal-status:
+          $ref: '#/components/schemas/LegalStatusSummary'
+        legacy-tna-cs13-record-type:
+          $ref: '#/components/schemas/LegacyTNACS13RecordType'
+        designation-of-edition:
+          type: string
+          description: Description identifying an edition to which a record (e.g. Map) belongs.
+        created:
+          type: object
+          oneOf:
+            - $ref: '#/components/schemas/TemporalInstant'
+            - $ref: '#/components/schemas/TemporalInterval'
+          description: The period of time or instant during which the record itself was created
+        covering:
```

Comment:
I'm not sure why we have create and covering? In the records I'm looking at we only have cat:created with a time interval, e.g.
```
cat:created [
    a                          time:ProperInterval ;
    dct:description "[1970]" ;
    time:hasBeginning  [
        a                            time:Instant ;
        time:inXSDDate  "1969-12-31Z"^^xsd:date
    ] ;
    time:hasEnd [
        a                            time:Instant ;
        time:inXSDDate  "1970-12-30Z"^^xsd:date
    ]
] 
```

Reply:
Records should have both a Created date and a Covering date. If we are not exposing the Covering date from the ETL then we are missing something (or perhaps it has been output with the wrong property name, e.g. `cat:created` rather than as the covering date) - we should check that!

Reply:
I'm a bit confused here. The covering dates currently go into a `cat:created` bNode but looking at the OCDM it shows both `cat:created` and `dct:temporal` as valid record description properties. We also have a `prov:generatedAtTime` property where we put the date the workflow is run. Of course, there are accumulation dates as well - but these are only at series or sub-series level.

Reply:
> I'm confused here

Yes... Me too it seems! I just re-checked a few things - The TNA-CS13 "Covering Dates" are now in Omega known as `cat:created`. This is the instant/period during which the record was created by the creating body.

The property `dct:temporal` can be used in Omega for the time instant/period that the record itself discusses within its content. I am not sure that we actually have such data available from ILDB for TNA's Paper Catalogue, but the property exists as a possible enhancement.

Reply:
So I think what you are saying here is that what we are seeing in the records as `cat:created` is correct and should be used to populate the `created` value within the schema. We don't currently have anything to put into the `covering` value but if we did it would appear in the records under `dct:temporal`.

Reply:
> cat:created is correct and should be used to populate the created value within the schema

Yes.

> We don’t currently have anything to put into the covering value but if we did it would appear in the records under dct:temporal

Exactly.